.env.example

```
@@ -0,0 +1,16 @@
+# Logging
+LOG_LEVEL=INFO
+
+# Dashboard
+DASHBOARD_HOST=127.0.0.1
+DASHBOARD_PORT=8050
+DASHBOARD_DEBUG=False
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The DASHBOARD_DEBUG key should go before the DASHBOARD_HOST key

(UnorderedKey)